src/commtrack.js

```diff
@@ -377,20 +399,22 @@ define([
             }
             this.__callOld();
         },
-        handleMugParseFinish: function (mug) {
-            this.__callOld();
-            if (!isTransaction(mug)) {
-                return;
+        parseSetValue: function (form, el, path) {
+            var mug = form.getMugByPath(path);
+            if (!mug) {
+                var basePath = path.replace(setValueDataRegex, "");
+                if (path !== basePath) {
+                    mug = form.getMugByPath(basePath);
+                    var setValue = _.find(setvalueData[mug.__className], function(sv) {
+                        return sv.path === path.match(setValueDataRegex)[1];
+                    }).attr;
```

Comment:
Could avoid the find for non-transaction widgets with paths that happen to match `setValueDataRegex` by moving the above 3 lines into the `if` block below.